### PR TITLE
reduce re-render on group card summary

### PR DESCRIPTION
**Pull request by Sbonelo01**
Related issues: [RENDER tidy]

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have merged the develop branch into my branch and fixed any merge conflicts
- [ x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x} My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by TheMaja — APPROVED** [review]

**Review by Vuyisanani — APPROVED** [review]

**Review by sheenarbw — CHANGES_REQUESTED** [review]
This code also doesn't address the issue. Please see the comments on your other recent PRs.

**Review by KhalidUmuzi — APPROVED** [review]

File: frontend/src/components/widgets/ActionLogCards.js [hunk 1]
```diff
@@ -35,14 +35,12 @@ const useStyles = makeStyles((theme) => ({
...
-    
-    ["@media (max-width:620px)"]: {// eslint-disable-line no-useless-computed-key
+    "@media (max-width:620px)": {
       fontSize: "0.7rem",
     },
   },
   title: {
-    
-    ["@media (max-width:620px)"]: {// eslint-disable-line no-useless-computed-key
+    "@media (max-width:620px)": {
```
Comment (sheenarbw) [inline frontend/src/components/widgets/ActionLogCards.js:43]: Why are you turning the es-lint warnings back on? Was that intentional or were you not sure what these comments are for?

**Review by sheenarbw — CHANGES_REQUESTED** [review]
The actual code looks fine, but I think you removed some useful comments

File: frontend/src/components/widgets/ActionLogCards.js (hunk 1 above)
Comment (Sbonelo01) [reply frontend/src/components/widgets/ActionLogCards.js:43]: I removed the square brackets and that fixed the warnings. https://eslint.org/docs/latest/rules/no-useless-computed-key

So I didn't think the comments were still necessary. Should I put them back?

Comment (KhalidUmuzi) [reply frontend/src/components/widgets/ActionLogCards.js:43]: This one works. I'm not sure why the other one is giving an issue

**Review by KhalidUmuzi — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**